### DISCUSSION TITLE
2 Fragen zur Verwendung

### DISCUSSION THREAD

**Discussion by Kiwi915** [pschatzmann/ESP32-A2DP#315] [Q&A] [answered]
Hallo,
 
Darf ich euch vieleicht bitte 2 Fragen stellen ?
 
1)
Ich habe bisher die "https://github.com/schreibfaul1/ESP32-audioI2S" auf meinem Wrover Board
getestet.
 
Hier habe ich wie folgt den externen DAC verdrahtet:
I2S_DOUT     25   // DIN connection
I2S_BCLK      27   // Bit clock
I2S_LRC        26   // Left Right Clock
 
Was wäre das bei dieser BT Lib als Receiver ?
.bck_io_num    = ?,
.ws_io_num     = ?,
.data_out_num  = ?,
 
Ich dachte ich frage lieber, weil ich nichts falsch machen möchte.
 
2)
Wisst ihr ob ich die ESP32-audioI2S mit dieser lib BT Sender kombinieren kann.
Ich würde gerne "einen Stream empfangen und über Bluetooth versenden"
Haben ihr eine Idee, oder geht das mit den Audio Tools ?
 
Liebe Grüße und vielen Dank schonmal 
 
Michaela

**Comment by pschatzmann** [comment, marked as answer]
Auf einem ESP32 kann nur entweder WIFI oder Bluetooth benutzt werden: Beides zusammen geht nicht!

- bck steht für bit clock und 
- ws für word select (= Left Right Clock)

**Comment by Kiwi915** [comment]
Hallo Danke,
aber ich verstehe in der Antwort immer noch nicht , welche Pins das in der Zuordnung bei mir sind.
Können sie mir bitte anhand meinen Pins zuordnen.
Lg

**Reply by pschatzmann** [reply]
Laut meiner Antwort: 
bck_io_num -> I2S_BCLK 27 // Bit clock   
ws_io_num -> I2S_LRC 26 // Left Right Clock

**Comment by Kiwi915** [comment]
Ok, dann danke.
Ich probiere es jetzt mal so.
Danke nochmal
